pkg/util/abstract/event.go

```diff
@@ -0,0 +1,104 @@
+package abstract
```

Comment:
util/abstract? An abstract event is scoped under the processor and we can't think of nuclio/nuclio as being processor centric just because that's the largest service at the moment. We will have quite a few large services and need to make sure not to pollute namespaces. As such, this abstract event should reside in somewhere like `pkg/processor/event.go` (`processor` package)